src/test/java/org/tuberlin/de/common/BaseJobGraphTest.java

```diff
@@ -55,9 +55,7 @@ public void setUp() throws Exception {
 
         ArrayList<String> alParents, alChildren;
 
-        aggParameters.put(TransformationAggregate.FIELD_KEY, 1);
-        aggParameters.put(TransformationAggregate.FUNCTION_KEY, TransformationAggregate.FUNCTION_TYPES.SUM);
-        TransformationAggregate aggregateComponent = new BaseTransformationAggregate(jobGraph, aggParameters);
```

Comment:
The jobGraph is referenced here while it is still not initialized (null). I moved this part to a point when the jobGraph is initialized. Then the test passes